### PR TITLE
#3422 More Circle colours

### LINKED ISSUE #3422: Vaccine module page fridge circles; More colours?
## Is your feature request related to a problem? Please describe.

Currently each fridge has a green circle rendered next to their name. The green doesn't actually mean anything, maybe we should do something with it?

## Describe the solution you'd like

Give the circle a purpose? Ideas:

- Orange circle if breached recently?
  - Red if currently breached?
- Change to different icons?
  - Over kill? The sensor info row of the fridge gives breach details already

## Implementation

TBD

## Describe alternatives you've considered

Do nothing. Ask a UI designer.

## Additional context

https://github.com/openmsupply/mobile/pull/3409#discussion_r566471691

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #3422 

## Change summary

- Moved the various components used in `SensorHeader` into its own folder
- Component `SensorIsInDangerCircle` .. maybe overkill - seemed nicer 🤷 

## Testing

- [ ] When a sensor is low battery, the circle is DANGER RED and flashing
- [ ] When a sensor is in breach, the circle is DANGER RED and flashing
- [ ] otherwise, it's green

### Related areas to think about

N/A

### CODE REVIEW

File: src/database/DataTypes/Sensor.js [hunk 1]
```diff
@@ -92,6 +92,22 @@ export class Sensor extends Realm.Object {
   get currentTemperature() {
     return this.mostRecentLog?.temperature;
   }
+
+  get isInHotBreach() {
+    const breach = this.mostRecentBreach;
+    const isOngoing = !!breach?.endTimestamp;
```
Comment (mark-prins) [inline src/database/DataTypes/Sensor.js:98]: is it ongoing if there is an end? I thought that it would be the other way round

**Review by mark-prins — APPROVED** [review]
😄  I like it a lot!

I think the logic of ongoing breach is backwards.. though you know this better than I so its probably my understanding that's out.

File: src/database/DataTypes/Sensor.js (hunk 1 above)
Comment (joshxg) [reply src/database/DataTypes/Sensor.js]: ah doh, nice! thanks 

0882239

**Comment by joshxg** [comment]
Have also added this commit: 
1195d55 

which was causing me a couple of issues

**Review by katherine-sussol — COMMENTED** [review]
Hehe, just wanted to say I like this animation, very eye-catching 👍

**Comment by joshxg** [comment]
haha cool - I thought it might be too much.....

**Pull request merged**